orchagent/portsorch.cpp

```diff
@@ -3727,7 +3744,7 @@ bool PortsOrch::initializePort(Port &port)
     }
 
     /* initialize port admin speed */
-    if (!getPortSpeed(port.m_port_id, port.m_speed))
+    if (!isAutoNegEnabled(port.m_port_id) && !getPortSpeed(port.m_port_id, port.m_speed))
     {
         SWSS_LOG_ERROR("Failed to get initial port admin speed %d", port.m_speed);
```

Comment:
If autoneg enabled, the error message "Failed to get initial port admin speed %d" is confusing.

Reply:
Acknowledge the log wording would be an issue. But let's not fix in a feature branch. let's fix it in master then cherry-pick.